app/views/help.mustache

```diff
@@ -49,7 +50,17 @@
   <h3 id="health">Health</h3>
   <p>
     The health score is based on static analysis of the package with
-    <code>dartanalyzer</code>.
+    <code>dartanalyzer</code>:
+  </p>
+  <ul>
+      <li>Each <i>error</i> reduces the health score by 25 percent <sup>(*)</sup>.</li>
```

Comment:
Maybe say "reduces the remaining health score by 25%" which sort of suggests the multiplicative behavior rather than a linear penalty. But it may be a little odd as it suggests an algorithm.